### PR TITLE
Add Program Identifier to Support Other Projects

### PR DESCRIPTION
**Pull request by jasondegraw**
The BuildingSync schema can be useful and leveraged for more types of project than just audits, and the current schema uses an audit element that can be confusing.

### CODE REVIEW

**Comment by nllong** [comment]
Implement this after #64 is merged down.

File: proposals/2018/Change_Audit_to_Project.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Implementation ##
+The proposed implementation is to add the elements marked with a '*' to the BuildingSync.xsd code:
+	>BuildingSync/
+		oPrograms/*
+			-Program/*
+				+ProgramDate*
+				+ProgramFundingSource*
+				+ProgramClassification*
+		oCampuses/Campus/
```
Comment (nllong) [inline proposals/2018/Change_Audit_to_Project.md]: This needs to be facilities/facilty

File: proposals/2018/Change_Audit_to_Project.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Note that this root element sequence refers to the one explained in Proposal #64.
+
+Steps:
+	a)Create an element named 'Programs' as a sibling of 'Campuses.'
```
Comment (nllong) [inline proposals/2018/Change_Audit_to_Project.md]: Facility

File: proposals/2018/Change_Audit_to_Project.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	b)Create an element named 'Program' as a child of 'Programs.'
+	c)Create the elements 'ProgramDate,' 'ProgramFundingSource,' and 'Program Classification' as children of 'Program.'
```
Comment (nllong) [inline proposals/2018/Change_Audit_to_Project.md]: Program Classification -> ProgramClassification

**Comment by aqwolfe** [comment]
@nllong there is not an example file using the program element, do you want me to modify an existing example? if so should that be a new example or just a new version of existing example?

**Comment by nllong** [comment]
Yes, i think modifying an existing example will work great. Thanks!

**Pull request merged**